### ISSUE TITLE
withExcludedLibraries doesn't work

### ISSUE THREAD

**Issue by roman-ku** [mikepenz/AboutLibraries#291]
Here is how I am using it:

                new LibsBuilder()
                        .withExcludedLibraries(
                                "Android-Iconics",
                                "FastAdapter"
                        )
                        .withLicenseShown(true)
                        .withActivityTitle(getString(R.string.about_licenses))
                        .withActivityTheme(R.style.ToolBarStyleAboutLibrary)
                        .start(getContext());

**Comment by mikepenz** [comment]
@roman-ku the with exclude and the other methods are all case sensitive.

**Comment by roman-ku** [comment]
Hmm, how do I know which name to put there? Like what exact library name do I use?

**Comment by mikepenz** [comment]
@roman-ku as for most libraries you can just find it out in its string file:
https://github.com/mikepenz/AboutLibraries/blob/develop/library/src/main/res/values/library_circleimageview_strings.xml#L3
(the name after "define_int_") Libraries which include the definition file are not included in the values folder so you have to check out the definition for those in their ressources. (Excluding is no documented feature as it is not great to exclude libraries which you have in your project) 
https://github.com/mikepenz/Android-Iconics/blob/develop/library-core/src/main/res/values/library_androidiconics_strings.xml#L19
https://github.com/mikepenz/FastAdapter/blob/develop/library-core/src/main/res/values/library_fastadapter_strings.xml#L4

**Comment by roman-ku** [comment]
Thanks, I will take a look at that.

>not great to exclude libraries which you have in your project

True. But I'm not sure why the `Android-Iconics` and `FastAdapter` libraries are showing up. I'm not using them in my app nor directly including them. I mean sure your library may be using them, but I don't think it's a legal requirement to include licenses for open source libraries which an open source library is based on or uses. That being said I'm not a lawyer so maybe I should be including them.

**Comment by mikepenz** [comment]
@roman-ku the `FastAdapter` is powering the list of the `AboutLibraries` as it is so lightweight I included it to simplify my live, and also to make things re-useable. 

As of the Android-Iconics. I had many requests of people wanting to use icons in the descriptions and buttons of the AboutLibraries. So I included the supprot for it (just the core)

And as those 2 libs are from me anyway. :D Go ahead and exclude them. I don't want to force people in showing the license even if it is just a lightweight sub-dependency :D
